plutus-contract-certification/src/Plutus/Contract/Test/Certification/Run.hs

```diff
@@ -103,11 +107,29 @@ makeLenses ''CertificationReport
 certResJSON :: CertificationReport m -> String
 certResJSON = unpack . encode
 
-data CertificationOptions = CertificationOptions { certOptNumTests :: Int
-                                                 , certOptOutput   :: Bool }
+data CertificationEvent = QuickCheckTestEvent (Maybe Bool)  -- ^ Nothing if discarded, otherwise test result
+                        | StartCertificationTask CertificationTask
+  deriving (Eq, Show)
+
+data CertificationTask = UnitTestsTask
+                       | StandardPropertyTask
+                       | DoubleSatisfactionTask
+                       | NoLockedFundsTask
+                       | NoLockedFundsLightTask
+                       | CrashToleranceTask
+                       | WhitelistTask
+                       | DLTestsTask
+  deriving (Eq, Show)
```

Comment:
Can we get `Enum` (reflecting the actual run order) and `Bounded`? That way if more stages are added I can still give an overall X/Y number until I update my source.

Reply:
Also can we get something like `renderCertificationTask :: CertificationTask -> Text/String` that's nicer for formatting to the user, again in the case where this repo updates before I've had a chance to update source on my end?

Reply:
If QC runs in more than one phase, can we get something like `hasQCTests :: CertificationTask -> Bool`?

Reply:
I don't think formatting for the end user is the responsibility of this code, and would be much better suited somewhere else.

Reply:
@UlfNorell OK, that's fine